src/main/java/storage/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package storage;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    private static final int INDEX_OF_HEAD_ELEMENT = 0;
+
+    public List<String> getInfoFromFile(File file) {
+        try {
+            List<String> parsedInfo = Files.readAllLines(file.toPath());
+            parsedInfo.remove(INDEX_OF_HEAD_ELEMENT);
```

Comment:
this is part of parsing logic
should be in separate service dedicated only to parsing